spec/scss_converter_spec.rb

```diff
@@ -230,4 +230,60 @@ def converter(overrides = {})
 
   end
 
+  context "importing from internal libraries" do
+    let(:internal_library) { source_dir("bower_components/jquery") }
+    let(:converter) { site.getConverterImpl(Jekyll::Converters::Scss) }
+
+    before(:each) do
+      FileUtils.mkdir_p(internal_library) unless File.directory?(internal_library)
+    end
+
+    after(:each) do
+      FileUtils.mkdir_p(internal_library) unless File.directory?(internal_library)
+    end
```

Comment:
Why do you create the internal library both before and after the test? Do you mean to remove in the `after` block here?